pontoon/sync/changeset.py

```diff
@@ -112,10 +118,12 @@ def execute_update_vcs(self):
         ).asymmetric().values_list('path', flat=True).distinct()
 
         for path in obsolete_entities_paths:
-            changed_resources.add(resources[path])
+            # Path can't be available if e.g. resource has been removed from the repository.
```

Comment:
This doesn't do any harm, but I'm also not sure it's neccessary given that we `update_resources()` before `update_entities()` in `core.py:sync_project()`.